test/data/wfs.py

```diff
@@ -0,0 +1,386 @@
+       xsi:schemaLocation="http://www.opengis.net/wfs/2.0
+                           http://schemas.opengis.net/wfs/2.0/wfs.xsd
+                           http://www.opengis.net/gml/3.2
+                           http://schemas.opengis.net/gml/3.2.1/gml.xsd">
+       <wfs:Insert>
+           <{username}:{layername}>
+               <{username}:wkb_geometry>
+                   <gml:MultiCurve srsName="urn:ogc:def:crs:EPSG::3857" srsDimension="2">
+                       <gml:curveMember>
+                           <gml:LineString>
+                               <gml:posList>3722077.1689 5775850.1007 3751406.9331 5815606.0102 3830548.3984 5781176.5357
+                                   3866350.4899 5774848.8358 3880796.9478 5743277.797 3897591.3679 5738418.6547
+                               </gml:posList>
+                           </gml:LineString>
+                       </gml:curveMember>
+                   </gml:MultiCurve>
+               </{username}:wkb_geometry>
+           </{username}:{layername}>
+       </wfs:Insert>
+    </wfs:Transaction>'''
+
+
+def get_wfs_insert_points_new_attr(username, layername, attr_names):
+    attr_xml = ' '.join([
+        f"<{username}:{attr_name}>some value</{username}:{attr_name}>"
+        for attr_name in attr_names
+    ])
+    return f'''<?xml version="1.0"?>
+<wfs:Transaction
+   version="2.0.0"
+   service="WFS"
+   xmlns:{username}="http://{username}"
+   xmlns:fes="http://www.opengis.net/fes/2.0"
+   xmlns:gml="http://www.opengis.net/gml/3.2"
+   xmlns:wfs="http://www.opengis.net/wfs/2.0"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://www.opengis.net/wfs/2.0
+                       http://schemas.opengis.net/wfs/2.0/wfs.xsd
+                       http://www.opengis.net/gml/3.2
+                       http://schemas.opengis.net/gml/3.2.1/gml.xsd">
+   <wfs:Insert>
+       <{username}:{layername}>
+           <{username}:wkb_geometry>
+               <gml:Point srsName="urn:ogc:def:crs:EPSG::3857" srsDimension="2">
+                   <gml:pos>1.27108004304E7 2548415.5977</gml:pos>
+               </gml:Point>
+           </{username}:wkb_geometry>
+           <{username}:name>New name</{username}:name>
+           <{username}:labelrank>3</{username}:labelrank>
+           {attr_xml}
+       </{username}:{layername}>
+   </wfs:Insert>
+   <wfs:Insert>
+       <{username}:{layername}>
+           <{username}:wkb_geometry>
+               <gml:Point srsName="urn:ogc:def:crs:EPSG::3857" srsDimension="2">
+                   <gml:pos>1.42108004308E7 2678415.5977</gml:pos>
+               </gml:Point>
+           </{username}:wkb_geometry>
+           <{username}:name>New name2</{username}:name>
+           <{username}:labelrank>4</{username}:labelrank>
+           {attr_xml}
+       </{username}:{layername}>
+   </wfs:Insert>
+</wfs:Transaction>'''
+
+
+def get_wfs_insert_points_new_attr_wfs10(username, layername, attr_names):
```

Comment:
now, when we use more WFS versions, what about naming methods in this file
`get_wfs<version>_...`, e.g. `get_wfs10_...`,  get_wfs20_...`.